assets/src/edit-story/components/fontPicker/pickerContainer.js

```diff
@@ -93,21 +139,30 @@ const SearchInput = styled.input.attrs({
   ['aria-autocomplete']: 'list',
   ['aria-owns']: 'editor-font-picker-list',
 })`
-  margin: 1px 1px 0;
-  padding: 6px 12px 6px 26px;
   width: 100%;
-  border-radius: 4px 4px 0 0;
-  background: ${({ theme }) => theme.colors.bg.v15};
-  border: none;
+  padding: 6px 20px 6px 30px;
+  border-radius: ${({ theme }) => theme.border.radius.default};
+  background: ${({ theme }) => theme.colors.input};
+  border: 1px solid ${({ theme }) => rgba(theme.colors.bg.white, 0.24)};
   color: ${({ theme }) => theme.colors.fg.white};
   font-size: ${({ theme }) => theme.fonts.input.size};
   line-height: ${({ theme }) => theme.fonts.input.lineHeight};
   font-weight: ${({ theme }) => theme.fonts.input.weight};
   font-family: ${({ theme }) => theme.fonts.input.family};
 
+  &::-ms-clear {
+    display: none;
+  }
+
+  &::-webkit-search-decoration,
+  &::-webkit-search-cancel-button,
+  &::-webkit-search-results-button,
+  &::-webkit-search-results-decoration {
+    appearance: none;
+  }
+
   &:focus {
-    outline: none;
-    border-color: ${({ theme }) => theme.colors.selection};
+    border-color: ${({ theme }) => theme.colors.accent.primary};
```

Comment:
Shouldn't it be `.secondary` here? Figma says blue, but the implementation is red. Also, the border width doesn't seem to match fully here.

| Figma | Implementation |
|-|-|
| ![Screen Shot 2020-09-02 at 22 39 14](https://user-images.githubusercontent.com/637548/92065280-6a5e3b00-ed6d-11ea-8dd2-a2ae2b96dc1c.png) | ![Screen Shot 2020-09-02 at 22 41 30](https://user-images.githubusercontent.com/637548/92065362-95e12580-ed6d-11ea-92d9-52d9f8bdd007.png) |

Reply:
hmph shouldn't be red. maybe something outdated left in there from when I was testing it out. `accent.primary` is definitely blue:
<img width="254" alt="Screen Shot 2020-09-03 at 10 07 25 AM" src="https://user-images.githubusercontent.com/35983235/92139683-5518f900-edcd-11ea-8872-d24512188218.png">

Reply:
Weird. definitely blue on my machine. remember I used tomato way back when I was debugging something around this. Maybe your branch is slightly out of date?

<img width="243" alt="Screen Shot 2020-09-03 at 10 24 37 AM" src="https://user-images.githubusercontent.com/35983235/92141503-c22d8e00-edcf-11ea-8f8c-2f940d6d8703.png">

Reply:
will update to `.secondary` tho so it matches the blue in `recommended` like in the designs. Worth noting that we're using `accent.primary` most places to indicate focus rn tho. Should maybe come to a consensus there so we don't have a bunch of different focus outlines

Reply:
Primary accent has changed color many times, but I think it's finally been settled on a nice purple now. But great that it's settled somewhere!